brigade/templates/national_advisory_council.html

```diff
@@ -0,0 +1,112 @@
+    </div>
+  </div>
+</section>
+
+<section class="slab border-top-slab">
+  <div class="grid-box">
+    <div class="width-seven-twelfths">
+      <h2>The current National Advisory Council</h2>
+
+      <p>There are nine NAC members (5 at-large and 4 regional), each are elected to serve a two year term with elections alternating each year.</p>
+
+      <p>They are:</p>
+
+      <ul>
+        <li>Jill Bjers, Chair (At-Large, Open Charlotte Brigade)</li>
+        <li>Jason Hibbets (At-Large, Open Raleigh Brigade)</li>
+        <li>Chris Alfano (At-Large, Code for Philly)</li>
+        <li>Nina Kin (At-Large, Hack for LA)</li>
+        <li>Emma Burnett (At-Large, Code for Maine)</li>
+        <li>Janet Michaelis (Midwest Region, Code for Dayton)</li>
+        <li>Laura Biediger (South Region, Code for Durham)</li>
+        <li>Ramy Kim (West Region, OpenOakland)</li>
+        <li>Dawn McDougall (Northeast Region, Code for Philly)</li>
+      </ul>
+
+      <p>You can reach the NAC by emailing <a href="mailto:NAC@codeforamerica.org">NAC@codeforamerica.org</a>.</p>
+
+    </div>
+
+    <div class="width-one-third shift-one-twelfth">
+      <img src="{{ asset_url_for('images/brigade-regions-2018.svg') }}" alt="Map of Brigade Regions" />
+
+      <p>&nbsp;</p>
+
+      <div class="message">
+        <div class="message__icon">
+          <i class="fas fa-info-circle"></i>
+        </div>
+        <div class="message__content">
+          <strong>Not sure which region your Brigade is in?</strong>
+          <a href="http://c4a.me/brigades" target="_blank">Check the Brigade Contact Directory</a>.
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="slab gray-bg-slab">
+  <div class="grid-box">
+    <div class="width-seven-twelfths">
+      <h2>National Advisory Council meetings</h2>
+
+      <p>
+        The National Advisory Council meets monthly to discuss their current work, upcoming events and Brigade network needs. The agenda is maintained and facilitated by the Chair and time is split between NAC members and CfA staff.
+      </p>
+      <p>
+        All agendas and notes are posted on <a href="https://discourse.codeforamerica.org" target="_blank">the Code for America Discourse</a> after the meeting.
+      </p>
+
+      <a href="https://discourse.codeforamerica.org/tags/c/from-nac/agenda" target="_blank" class="button button-outline button-linkout">View Meeting Agendas</a>
+
+      <p>Additionally, the NAC has two in-person meetings a year -- once at Summit and also at Brigade Congress.</p>
+    </div>
+  </div>
+</section>
+
+<section class="slab">
+  <div class="grid-box">
+    <div class="width-seven-twelfths">
+      <h2>Previous National Advisory Councils</h2>
+
+      <h3>2016-2017</h3>
```

Comment:
I would use an emdash here: `2016—2017`

Reply:
The Em-dash looked huge, so I split the difference and went with an en-dash instead. [Punctuation pedants](https://getitwriteonline.com/articles/en-dashes-em-dashes/) also indicate that en-dashes indicate ranges, so this seems legit.

Reply:
Good call!